apps/etterlatte-migrering/src/main/kotlin/migrering/LyttPaaDistribuerBrevRiver.kt

```diff
@@ -0,0 +1,40 @@
+package no.nav.etterlatte.migrering
+
+import no.nav.etterlatte.rapidsandrivers.migrering.Migreringshendelser
+import no.nav.helse.rapids_rivers.JsonMessage
+import no.nav.helse.rapids_rivers.MessageContext
+import no.nav.helse.rapids_rivers.RapidsConnection
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import rapidsandrivers.migrering.ListenerMedLoggingOgFeilhaandtering
+import java.util.UUID
+
+internal class LyttPaaDistribuerBrevRiver(
+    rapidsConnection: RapidsConnection,
+    private val pesysRepository: PesysRepository,
+) :
+    ListenerMedLoggingOgFeilhaandtering(Migreringshendelser.BREV_SENDT) {
+    private val logger: Logger = LoggerFactory.getLogger(this::class.java)
+
+    init {
+        initialiserRiver(rapidsConnection, "BREV:DISTRIBUERT") {
```

Comment:
Pirk, men BrevEventTypes.DISTRIBUERT.toString()

Reply:
Så på denne jeg også, men isåfall burde/må `BrevEventTypes` trekkes ut i eget modell-lib, siden den ligger i `etterlatte-brev-api`.